### PR TITLE
Add `IS_CE` to constants

### PR DESCRIPTION
**Pull request by yunline**
Issue: #1845

### CODE REVIEW

File: buildconfig/stubs/pygame/__init__.pyi [hunk 1]
```diff
@@ -52,12 +51,6 @@ from .math import Vector2 as Vector2, Vector3 as Vector3
 from .cursors import Cursor as Cursor
 from .bufferproxy import BufferProxy as BufferProxy
 from .mask import Mask as Mask
-from ._debug import print_debug_info as print_debug_info
```
Comment (yunline) [inline buildconfig/stubs/pygame/__init__.pyi:55]: Why are these changed???

**Review by oddbookworm — CHANGES_REQUESTED** [review]
In my testing, this worked great. `getattr(pygame, "IS_CE", False)` returned `True`. I think docs should be added for this feature however, which is the reason for the change request. Probably best to document it in https://github.com/pygame-community/pygame-ce/blob/main/docs/reST/ref/pygame.rst IMO

**Comment by MyreMylar** [comment]
This also seems to be crashing the CI in the sphinx docs.

Perhaps we could just do a:

```
DEC_CONSTS(IS_CE, 1);
```

instead?

**Comment by yunline** [comment]
> This also seems to be crashing the CI in the sphinx docs.
> 
> Perhaps we could just do a:
> 
> ```
> DEC_CONSTS(IS_CE, 1);
> ```
> 
> instead?

I doubt if the crash was caused by IS_CE.
And IS_CE should be a bool but DEC_CONSTS genetates an integer constant.

**Review by MyreMylar — APPROVED** [review]
Tested with:

```
import pygame


if getattr(pygame, "IS_CE", False):
    print("Running pygame CE")
else:
    print("Not running pygame CE")
```

And it works as it should.

**Review by oddbookworm — APPROVED** [review]
LGTM!

**Pull request merged**